examples/01_board_and_wall.py

```diff
@@ -1,5 +1,4 @@
 
-import examples_includes
```

Comment:
Why did you remove that line?

Reply:
if it's not used i think there is no need.
flake8 will trhow an errror for unused imports

Reply:
Actually it is only 2 lines to include ../ into the path so when you run the examples from the example directory without installing the library first, it still work.
Maybe it's the wrong way of doing it.